test/albums.test.js

```diff
@@ -12,6 +12,14 @@ const responseAlbumsList = [
   }
 ];
 
+const albumPhotos = {
```

Comment:
this object is already define in resquest-promise.js file

Reply:
moved to  utils.test.js